style.css

```diff
@@ -1,4 +1,347 @@
-* {
-  padding: 0;
-  margin: 0;
-}
+    * {
+        padding: 0;
+        margin: 0;
+        box-sizing: border-box;
+    }
+    
+    #pagemidcont {
+        display: flex;
+        justify-content: space-between;
+        width: 930px;
+        margin: 20px auto;
+    }
+    
+    #topicscont {
+        display: flex;
+        flex-direction: column;
+    }
+    
+    #asidepartcont {
+        display: flex;
+        flex-direction: column;
+    }
+    
+    .topic1 {
+        width: 610px;
+        height: 600px;
+    }
+    
+    .newsletter,
+    .category,
+    .popup1,
+    .popup2 {
```

Comment:
These four classes you can exchange them with one class because all of them have the same style. and you don't need them in another place. except` .popup1` you can keep it because you give it more styles.

Reply:
okay